dumb-init.c

```diff
@@ -128,6 +128,10 @@ int main(int argc, char *argv[]) {
         }
 
         execvp(argv[1], &argv[1]);
+
+        // if this point is reached, exec failed, so we should exit nonzero
+        DEBUG("execvp failed in child; errno=%d %s\n", errno, strerror(errno));
+        exit(2);
     } else {
```

Comment:
The user needs more feedback in this case.
This isn't a debugging situation.
For example, bash prints:

```
$ wattttt foo
-bash: wattttt: command not found
```